public/CHANGELOG.md

```diff
@@ -44,16 +57,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - merged into 0.1.19
 
-## [0.1.19] 2024-11-18
```

Comment:
any particular reason why this was deleted

Reply:
not sure!

Reply:
this file was synced from the main changelog so this is now correct.